remote-messaging/remote-messaging-api/src/main/java/com/duckduckgo/remote/messaging/api/RemoteMessage.kt

```diff
@@ -78,10 +90,11 @@ sealed class Content(val messageType: MessageType) {
     }
 }
 
-sealed class Action(val actionType: String, open val value: String) {
+sealed class Action(val actionType: String, open val value: String = "", open val additionalParameters: JsonAdditionalParameters? = null) {
```

Comment:
I don't understand the `JsonAdditionalParameters`. It's also increasing the API surface.
Why can't that be either a `String` or even better, be inside the `value`?

I am also a bit itchy when we add default values to required parameters (ie. `value`). If it's a require parameter we should no add a default value so that users of that method think about what value they want to pass.
If it's an optional value, it should be typed as `String?`
Can we undo the change `val value: String = ""` ?